.appveyor.yml

```diff
@@ -43,18 +42,36 @@ build_script:
     # Build and install miri
     - cargo build --release --all-features --all-targets --locked
     - cargo install --all-features --force --path . --locked --offline
-    # Get ourselves a MIR-full libstd, and use it henceforth
-    - cargo miri setup
-    - set MIRI_SYSROOT=%USERPROFILE%\AppData\Local\rust-lang\miri\cache\HOST
 
 test_script:
     - set RUST_TEST_NOCAPTURE=1
     - set RUST_BACKTRACE=1
-    # Test miri
+    # Test host miri: 32bit Windows
+    - cargo miri setup
+    - set MIRI_SYSROOT=%USERPROFILE%\AppData\Local\rust-lang\miri\cache\HOST
+    - cargo test --release --all-features --locked
+    - cd test-cargo-miri
+    - '"C:\msys64\mingw64\bin\python3.exe" run-test.py'
+    - cd ..
+    - set MIRI_SYSROOT=
```

Comment:
This here seems to not have the effect of deleting the env var, even though that's what [the internet](https://stackoverflow.com/questions/7219672/how-to-undefine-a-ms-dos-variable) says I should do.